proto/storage/virtual_machine.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package storage;
+
+import "google/protobuf/timestamp.proto";
+import "storage/image.proto";
+
+message VirtualMachine {
+  string id = 1; // @gotags: search:"Virtual Machine ID,store,hidden" sql:"pk"
+  string namespace = 2; // @gotags: search:"Namespace,store"
+  string name = 3; // @gotags: search:"Name,store"
+  string cluster_id = 4; // @gotags: search:"Cluster ID,hidden,store"  sql:"type(uuid)"
+  string cluster_name = 5;
+  map<string, string> facts = 6;
+
+  VirtualMachineScan scan = 7; // @gotags: policy:"Virtual Machine Scan"
+
+  google.protobuf.Timestamp last_updated = 8; // @gotags: search:"Last Updated,hidden"
+
+  enum Note {
+    MISSING_METADATA = 0;
+    MISSING_SCAN_DATA = 1;
+    MISSING_SIGNATURE = 2;
+    MISSING_SIGNATURE_VERIFICATION_DATA = 3;
+  }
+  repeated Note notes = 9;
+}
+
+// TODO: Consider cleaning this up. It doesn't have to perfectly match the image scan message.
+message VirtualMachineScan {
+  string scanner_version = 1;
+  google.protobuf.Timestamp scan_time = 2; // @gotags: search:"Virtual Machine Scan Time,store"
+  repeated EmbeddedImageScanComponent components = 3; // @gotags: sql:"-"
```

Comment:
Can you put a TODO here so we DO NOT build tables and ship with this as EmbeddedImageScanComponent without considerable thought?  My gut reaction is we probably want to really look and evaluate if we want to re-use that object as is or make adjustments in VM specific version.